src/test/components/ProfileViewer.test.js

```diff
@@ -37,12 +42,23 @@ describe('ProfileViewer', function() {
       .mockImplementation(() => ctx);
   });
 
-  it('calculates the full timeline height correctly', () => {
+  function setup() {
     // WithSize uses requestAnimationFrame
     const flushRafCalls = mockRaf();
-    const store = storeWithProfile(getProfileWithNiceTracks());
 
-    render(
+    const store = blankStore();
+    store.dispatch(
+      updateUrlState(
+        stateFromLocation({
+          pathname: '/from-addon',
+          search: '',
+          hash: '',
+        })
+      )
+    );
```

Comment:
This is made necessary because the new code throws an error when the datasource is "none" (which makes sense).